js/src/personal/bryan/TimeDisplay.tsx

```diff
@@ -18,7 +18,7 @@ export function TimeDisplay() {
     'Dec',
   ]
   const [date, setDate] = useState(new Date())
-
+  // useEffect is used here to allow setsDate to rerun every second
```

Comment:
In comments, you generally want to be putting the "Why" and not the "How".

Your comment directly mirrors the code and can be read from the code.
Something like
` // Refresh the time every second to keep it at the current time.`
tells you why the useEffect exists.